### ISSUE TITLE
argument parsing error in msvc_build_libraries.py for windows installation

### ISSUE THREAD

**Issue by TheCalculus** [c3lang/c3c#1050]
when running `msvc_build_libraries.py` to complete the windows installation, the following error occurs:
```sh
> python msvc_build_libraries.py --msvc-version 14.37 --sdk_path-version 22621
Traceback (most recent call last):
  File "C:\Users\thecalculus\c3-windows\c3-windows-Release\msvc_build_libraries.py", line 114, in <module>
    sdk_ver = args.sdk_version or max(sorted(sdk_path.keys()))
              ^^^^^^^^^^^^^^^^
AttributeError: 'Namespace' object has no attribute 'sdk_version'. Did you mean: 'msvc_version'?
```
this happens because of line 71 in the script:
```python
67 ap = argparse.ArgumentParser()
...
71 ap.add_argument("--sdk_path-version", help="Get specific Windows SDK version")
72 args = ap.parse_args()
```
...where replacing `--sdk_path-version` with just `--sdk-version` should resolve this problem

tldr;
```python
67 ap = argparse.ArgumentParser()
68 ap.add_argument("--show-versions", const=True, action="store_const", help="Show available MSVC and Windows SDK versions")
69 ap.add_argument("--accept-license", const=True, action="store_const", help="Automatically accept license")
70 ap.add_argument("--msvc-version", help="Get specific MSVC version")
71 ap.add_argument("--sdk-version", help="Get specific Windows SDK version") # replaced sdk_path-version with sdk-version
72 args = ap.parse_args()
```

**Comment by lerno** [comment]
Could you make a pull request for a fix?

**Comment by TheCalculus** [comment]
fix #1052
